### PR TITLE
Spring boot example

### PR DESCRIPTION
**Pull request by wallies**
added to roadmap as well

### CODE REVIEW

**Comment by pmbauer** [comment]
If this commit is merged, the listener protocols here ought to be changed to http.
https://github.com/Capgemini/Apollo/commit/96b2716d35851afd53969d233b8eb9b86c7219c2#diff-ee33aa395b84461181b557096d97ed52R8
See https://github.com/Capgemini/Apollo/pull/281#issuecomment-109124338

**Comment by andrewharmellaw** [comment]
Nice.

File: examples/spring-boot/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+### For Vagrant only
+
+If you are deploying this example in the Vagrant environment, simply add the following entry to your host machine in ```/etc/hosts```
+
+```
+172.31.1.14 spring.example.com
+```
+
+Where ```172.31.1.14``` is the Slave IP address configured in ```vagrant.yml```
+If you access ```drupal.example.com``` through your browser that will hit the Haproxy container
```
Comment (andrewharmellaw) [inline examples/spring-boot/README.md]: Should the "drupal" be changed to "spring" here?

**Comment by tayzlor** [comment]
This PR is a bit confused should actually be 3 separate ones (which is what i'd like to see) - 

- 1 for the roadmap update
- 1 for the spring boot plugin
- the ELB fixes should continue in the already open PR over here #281

**Pull request merged**